### PR TITLE
Adding pause/go button.

### PR DESCRIPTION
**Pull request by mithro**
Images:
- image
- image

### CODE REVIEW

File: test/bootstrap.js [hunk 1]
```diff
@@ -835,13 +849,22 @@ TestTimeline.prototype.autorun = function() {
 // Capture the real requestAnimationFrame so we can run in 'real time' mode
 // rather than as fast as possible.
 var raf = window.requestAnimationFrame;
-var raf_t0 = null;
+var raf_now = Date.now;
+
+var raf_t0 = null;    // Time the page loaded
+var raf_pause = null; // Time at which we paused raf
```
Comment (dstoc) [inline test/bootstrap.js]: If we want to match the spec/polyfill, this is called pauseStartTime

File: test/bootstrap.js [hunk 2]
```diff
@@ -835,13 +849,22 @@ TestTimeline.prototype.autorun = function() {
...
+var raf_delta = 0;    // Amount we have been stopped for.
```
Comment (dstoc) [inline test/bootstrap.js]: and this is timeDrift

File: test/bootstrap.js [hunk 3]
```diff
@@ -865,6 +888,24 @@ function testharness_raf(ts) {
...
+    raf_pause = null;
+
+    raf(testharness_raf);
+  }
+}
+
+function testharness_raf_pause() {
+  if (raf_pause != null) {
+    return;
+  } else {
```
Comment (dstoc) [inline test/bootstrap.js]: else not needed since early return, or flip the condition

File: test/bootstrap.js [hunk 4]
```diff
@@ -835,13 +849,22 @@ TestTimeline.prototype.autorun = function() {
...
+var raf_t0 = null;    // Time the page loaded
+var raf_pause = null; // Time at which we paused raf
+var raf_delta = 0;    // Amount we have been stopped for.
 function testharness_raf(ts) {
   if (raf_t0 === null) {
     raf_t0 = ts;
   }
 
-  var t = ts - raf_t0;
+  // Have we just unpaused and thus need to adjust the delta.
```
Comment (dstoc) [inline test/bootstrap.js]: Isn't this just checking whether we are paused or not? I'm not sure how it applies to "just unpaused" ?

File: test/bootstrap.js (hunk 1 above)
Comment (mithro) [inline test/bootstrap.js]: Done.

File: test/bootstrap.js (hunk 2 above)
Comment (mithro) [inline test/bootstrap.js]: Done.

File: test/bootstrap.js (hunk 3 above)
Comment (mithro) [inline test/bootstrap.js]: Done.

File: test/bootstrap.js (hunk 4 above)
Comment (mithro) [inline test/bootstrap.js]: Stale comment. Fixed.

**Comment by mithro** [comment]
PTAL.

**Pull request merged**